causalpy/utils.py

```diff
@@ -16,6 +17,32 @@ def _series_has_2_levels(series: pd.Series) -> bool:
     return len(pd.Categorical(series).categories) == 2
 
 
+def compute_bayesian_tail_probability(posterior, x) -> float:
+    """
+    Calculate the probability of a given value being in a distribution defined by the posterior,
+
+    Args:
```

Comment:
Same comment as above about the parameters and returns blocks not rendering properly in when you build and check the docs locally.